dimod/binary/binary_quadratic_model.py

```diff
@@ -899,11 +916,8 @@ def contract_variables(self, u, v):
         corresponding interactions of ``u``.
 
         Args:
-            u (variable):
-                Variable in the binary quadratic model.
-
-            v (variable):
-                Variable in the binary quadratic model.
+            u (variable): Variable in the binary quadratic model.
```

Comment:
Don't need `(variable)`, should instead update the type annotations.

Reply:
Changed definition to use type annotations